plugin-config/src/ConfigurationForm.jsx

```diff
@@ -6,6 +6,12 @@ import Form from "react-jsonschema-form";
 
 export default(props) => {
   const schema = JSON.parse(JSON.stringify(props.plugin.schema))
+  var uiSchema = {}
+
+  if ( typeof props.plugin.uiSchema !== 'undefined' ) {
+    uiSchema['configuration'] = JSON.parse(JSON.stringify(props.plugin.uiSchema))
```

Comment:
touche, I probably had some idea there but for the life of me can't recall what it was...